tests/testthat/_snaps/forge-formula.md

```diff
@@ -4,79 +4,90 @@
       forge(example_train2, x1$blueprint, outcomes = TRUE)
     Condition
       Error in `validate_column_names()`:
-      ! The following required columns are missing: 'fac_1'.
+      ! The required column `fac_1` is missing.
 
 ---
 
     Code
       forge(example_train2, x2$blueprint, outcomes = TRUE)
     Condition
       Error in `validate_column_names()`:
-      ! The following required columns are missing: 'fac_1'.
+      ! The required column `fac_1` is missing.
 
 # new_data can only be a data frame / matrix
 
     Code
       forge("hi", x1$blueprint)
     Condition
       Error in `forge()`:
-      ! The class of `new_data`, 'character', is not recognized.
+      ! No `forge()` method provided for a string.
 
 ---
 
     Code
       forge("hi", x2$blueprint)
     Condition
       Error in `forge()`:
-      ! The class of `new_data`, 'character', is not recognized.
+      ! No `forge()` method provided for a string.
 
 # missing predictor columns fail appropriately
 
     Code
       forge(example_train[, 1, drop = FALSE], x1$blueprint)
     Condition
       Error in `validate_column_names()`:
-      ! The following required columns are missing: 'num_2'.
+      ! The required column `num_2` is missing.
 
 ---
 
     Code
       forge(example_train[, 1, drop = FALSE], x2$blueprint)
     Condition
       Error in `validate_column_names()`:
-      ! The following required columns are missing: 'num_2'.
+      ! The required column `num_2` is missing.
 
 ---
 
     Code
       forge(example_train[, 3, drop = FALSE], x1$blueprint)
     Condition
       Error in `validate_column_names()`:
-      ! The following required columns are missing: 'num_1', 'num_2'.
+      ! The required columns `num_1` and `num_2` are missing.
 
 ---
 
     Code
       forge(example_train[, 3, drop = FALSE], x2$blueprint)
     Condition
       Error in `validate_column_names()`:
-      ! The following required columns are missing: 'num_1', 'num_2'.
+      ! The required columns `num_1` and `num_2` are missing.
 
 # novel predictor levels are caught
 
     Code
       xx1 <- forge(new, x1$blueprint)
     Condition
       Warning:
-      Novel levels found in column 'f': 'e'. The levels have been removed, and values have been coerced to 'NA'.
+      Novel level found in column f: "e".
+      i The level has been removed, and values have been coerced to "NA".
 
 ---
 
     Code
       xx2 <- forge(new, x2$blueprint)
     Condition
       Warning:
-      Novel levels found in column 'f': 'e'. The levels have been removed, and values have been coerced to 'NA'.
+      Novel level found in column f: "e".
+      i The level has been removed, and values have been coerced to "NA".
+
+---
+
+    Code
+      xx3 <- forge(new_multiple, x3$blueprint)
+    Condition
+      Warning:
+      Novel levels found in column f: "e" and "f".
+      i The levels have been removed, and values have been coerced to "NA".
```

Comment:
Can you change this to `<NA>`? We know what it means but, since they are character, it's not correct.

Reply:
resolved in [986f778](https://github.com/tidymodels/hardhat/pull/272/commits/986f7786bbce4914c2fa46626e99ad8595a60c53)